doc/release-notes-5493.md

```diff
@@ -0,0 +1,5 @@
+Masternode Reward Changes
+------------------------
+
+Since v19, EvoNodes are getting paid 4 block in a row.
+Once Masternode Reward Location Reallocation activates, EvoNodes will be paid as usual: one block in a row. 
```

Comment:
Maybe expand to something like this?
```suggestion
Since v19, EvoNodes are paid 4 blocks in a row.
Once Masternode Reward Location Reallocation activates, all masternodes, including EvoNodes, will be paid as usual by Core in a single block. The remaining rewards for EvoNodes will be paid out by Platform.
```